config.html

```diff
@@ -11,7 +11,7 @@
     const RELOAD = 0;
```

Comment:
I'm wondering maybe we should move javascript/css code to separate files from config.html as it's starting to get messy in here.

Reply:
I try something about it here: https://github.com/SubZeroPL/cs-rin-ru-enhanced-mod/tree/Separate-config-file
This adds this line to the web page: <script src="https://raw.githubusercontent.com/SubZeroPL/cs-rin-ru-enhanced-mod/Separate-config-file/config.js" type="text/javascript" id="lr1oml1c.v"></script>
with the config.js file containing all the javascript code. But the rest of the code can't access the functions that come from config.js.
I can't find the problem.